src/styles/PostBlock.scss

```diff
@@ -0,0 +1,61 @@
+$BACKGROUND-WHITE: #ffffff;
```

Comment:
maybe make the variable just `$background-color` instead. Since the value already tells us that it's white and in its current state, it we change value then we have to change the variable name too.

Reply:
done